processing/data_collection/gazette/spiders/es_vila_velha.py

```diff
@@ -0,0 +1,56 @@
+import dateparser
+
+from datetime import datetime, date
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+from scrapy import FormRequest, Request
+import os
+
+
+class VilaVelhaSpider(BaseGazetteSpider):
+    name = "es_vila_velha"
+    allowed_domains = ["www.vilavelha.es.gov.br"]
+
+    TERRITORY_ID = "3205200"
+    GAZETTE_URL_CSS = "td:last-child a::attr(href)"
+    GAZETTE_DATE_CSS = "td:nth-child(2) span b::text"
+    GAZETTE_ISSUE_CSS = "td:nth-child(3) span b::text"
+    JAVASCRIPT_POSTBACK_REGEX = r"javascript:__doPostBack\('(.*)',''\)"
+
+    # this is the first date which has gazettes available (edition number 1)
```

Comment:
I don't think this comment is necessary since it is a project pattern. Do you have any thoughts about this?

Reply:
I just wanted to make clear that there was no point setting the default date to anything before 2016-07-01, but on the other hand I noticed that if we try dates like 2010-01-01 it gets _much_ slower to get a response back, probably due to the way the database query works over there in Vila Velha. But maybe that's what should have been in the comment if I want to keep some :)

Reply:
Hm... Understood :) Yeah, but since dates before the spider's `start_date` shouldn't be run, we shouldn't have this kind of problem and that kind of comment wouldn't be necessary too.